### PR TITLE
Bnnz 1 as a guest i want to be able to login

### PR DESCRIPTION
**Pull request by TCLaurentiu**

### CODE REVIEW

File: bananza-frontend/styles/globals.css [hunk 1]
```diff
@@ -11,6 +21,145 @@ a {
...
+form div>span {
+  display:table;
+  margin:5px;
+}
+
+label input{
+  display:table;
+}
+
+input {
```
Comment (serbanlex) [inline bananza-frontend/styles/globals.css:72]: Can we do something about the input changing color to plain white on click (focus)? It kinda breaks your nice design, i think it shouldn't do anithing to the background of it or just add a shade or something, not white as it's now.
I think you can find something about this [here](https://stackoverflow.com/questions/66073842/how-do-i-stop-the-input-form-to-change-background-color-and-text-color-when-i-in)

File: bananza-frontend/components/general/nav.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import Link from "next/link";
```
Comment (serbanlex) [inline bananza-frontend/components/general/nav.js:1]: I would add some more lateral padding to the buttons (Home, Profile etc.). Maybe something to highlight them too (some background, some outline?) - it's white text over some supposedly colored background.

Also, the Channel button may be obsolete (we agreed the profile is the same thing as a channel). Maybe we can call the profile as a channel on the page?

File: bananza-frontend/components/reports/latestReports.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import Report from "./report.js";
```
Comment (serbanlex) [inline bananza-frontend/components/reports/latestReports.js:1]: Some padding problems here too, to be reviewed.

File: bananza-frontend/components/videos/topfive.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        <div className={styles.topFiveVideosContainer}>
+        {
+        props.videos.map((video, key) => (
+           <VideoCard video={video} key={key}/>
+        ))
+        }
+        </div>
+        </div>
+    );
+}
```
[the remark this answers is not in the document]
Comment (TCLaurentiu) [reply bananza-frontend/components/videos/topfive.js]: Should now be fixed.

File: bananza-frontend/components/general/nav.js (hunk 2 above)
Comment (TCLaurentiu) [reply bananza-frontend/components/general/nav.js:1]: Channel button has been removed.  Navbar has been improved somewhat, and may be modified more in the future.

File: bananza-frontend/styles/globals.css (hunk 1 above)
Comment (TCLaurentiu) [reply bananza-frontend/styles/globals.css:72]: Good idea! Improved this a bit, will probably be modified more in the future.

**Pull request merged**